lib/errorServer.js

```diff
@@ -51,25 +61,35 @@ function runErrorServer (error) {
       res.end(knownPaths[req.url].contents)
       return
     }
-    if (req.url.startsWith('/plugin-assets')) {
-      res.setHeader('Content-Type', fileExtensionsToMimeTypes[req.url.split('.').at(-1)] || 'text/plain')
-      const urlParts = req.url.split('/').slice(2)
-      const pluginName = urlParts.shift()
-      let filePath
-      if (pluginName === 'govuk-frontend') {
-        filePath = path.join(getInternalGovukFrontendDir(), ...urlParts)
-      } else {
-        filePath = path.join(process.cwd(), 'node_modules', pluginName, ...urlParts)
+
+    for (const knownRoute of [
+      '/manage-prototype/dependencies/',
+      '/plugin-assets/'
+    ]) {
+      if (!req.url.startsWith(knownRoute)) {
+        continue
       }
+
+      const filePath = req.url.split(knownRoute).at(-1)
+      const fileExtension = filePath.split('.').at(-1)
+
+      res.setHeader('Content-Type', fileExtensionsToMimeTypes[fileExtension] || 'text/plain')
+
+      // Route GOV.UK Frontend to internal package
+      const modulesDir = filePath.startsWith('govuk-frontend/')
```

Comment:
@nataliecarey I've kept the existing behaviour, i.e. always "route to internal"

But probably don't need to do that anymore for `/plugin-assets`?

```suggestion
      const modulesDir = filePath.startsWith('govuk-frontend/') && knownRoute.startsWith('/manage-prototype')
```

Reply:
> But probably don't need to do that anymore for `/plugin-assets`?

Correct, we should really be using the `/manage-prototype/dependencies/` URLs instead of the `/plugin-asset` URLs.

Reply:
(which you've changed in this PR) 🎉

Reply:
Nice, I'll add my suggestion 👍